### PR TITLE
ZEPPELIN-334 Add documentation for Apache Lens Interpreter

### PR DESCRIPTION
**Pull request by AhyoungRyu**
[ZEPPELIN-334](https://issues.apache.org/jira/browse/ZEPPELIN-334?jql=project%20%3D%20ZEPPELIN%20AND%20text%20~%20%22lens%22) Add documentation for Apache Lens Interpreter

Since the last PR that I opened few days ago has some problems, so I create new one. 
I read all comments in the last one and edited below files.

### CODE REVIEW

File: docs/docs/interpreter/lens.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      <td></td>
+  </tr>
+  <tr>
+      <td>lens.server.base.url</td>
+      <td>http://hostname:port/lensapi</td>
+      <td>The base url for the lens server. you have to edit "hostname" and "port" that you may use(ex. http://0.0.0.0:9999/lensapi)</td>
+   </tr>
+   <tr>
+      <td>lens.session.cluster.user </td>
+      <td>defalut</td>
```
Comment (felixcheung) [inline docs/docs/interpreter/lens.md]: 'default'

File: docs/docs/interpreter/lens.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  </tr>
+  <tr>
+      <td>lens.client.dbname</td>
+      <td>default</td>
+      <td>The database schema name</td>
+  </tr>
+  <tr>
+      <td>lens.query.enable.persistent.resultset</td>
+      <td>false</td>
+      <td></td>
```
Comment (felixcheung) [inline docs/docs/interpreter/lens.md]: could you add a description on this?

File: docs/docs/interpreter/lens.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  </tr>
+  <tr>
+      <td>zeppelin.lens.maxResult</td>
+      <td>1000</td>
+      <td>Max number of rows to display</td>
+  </tr>
+  <tr>
+      <td>zeppelin.lens.maxThreads</td>
+      <td>10</td>
+      <td></td>
```
Comment (felixcheung) [inline docs/docs/interpreter/lens.md]: could you add a description on this?

**Comment by AhyoungRyu** [comment]
Thank you for your reviews! 
I modified the typo and add the description that you mentioned.

**Comment by felixcheung** [comment]
looks good

**Comment by anthonycorbacho** [comment]
Great job @AhyoungRyu

**Comment by Leemoonsoo** [comment]
Great job. LGTM

**Pull request closed without merging**